bundle/README.md

```diff
@@ -0,0 +1,54 @@
+# About the Sail Operator
+
+The Sail Operator is based on the open source Istio project. The Sail implementation pulls its code from the upstream Istio master repository with no changes to the codebase.
+
+# Prerequisites
+
+You have deployed a cluster on OpenShift Container Platform 4.13 or later.
+
+# Installing the Sail Operator
+
+1. Create the project where Istio is going to be deployed:  
```

Comment:
Steps 1 and 2 are actually part of the `Deploying the control plane` section

Reply:
These were moved. I walked through the steps this way just to see it for myself. Thank you for helping to hone the procedure.